test/unit/ux/spinner.test.ts

```diff
@@ -28,7 +28,7 @@ describe('Spinner', () => {
       const spinner = new Spinner(true);
       spinner.start('Doing things');
       spinner.stop('Finished');
-      expect(writeStub.firstCall.args).to.deep.equal(['stderr', 'Doing things...\n']);
+      expect(writeStub.firstCall.args).to.deep.equal(['stderr', 'Doing things...']);
```

Comment:
If you run these tests individually with`yarn mocha test/unit/ux/spinner.test.ts` they expect the `\n` but if you run it from `yarn test` (which uses `wireit`) it doesn't think the `\n` should be there.

There's a check in oclif/core for `process.stderr.isTTY`. If it's undefined or false, it uses the `simple` spinner (`src/cli-ux/action/simple.ts`), otherwise it uses the default spinner (`src/cli-ux/action/spinner.ts`)

The defaults spinner passes in the message with the `\n`, whereas the simple spinner doesn't.

That being said, I'm not sure which (if any) change in v3 caused this test to start failing. As far as I can tell none of the logic around that part of the code base has changed